src/test/integration/bearer.test.ts

```diff
@@ -122,4 +123,14 @@ describe('Bearer authentication middleware', () => {
         expect(body.email).toBe(email);
         expect(body).toMatchSnapshot();
     });
+    test('Can refresh token', async () => {
+        const options = getAuthistOptions();
+        const serverUrl = startServer(options);
+        const { credentials } = await emailPasswordProvider.signInWithEmailAndPassword(options)(email, password);
+        const { body } = await got<User & { password?: string }>(`${serverUrl}/refresh-token`, {
+            responseType: 'json',
+            searchParams: { refreshToken: credentials.refreshToken },
```

Comment:
Shouldn't we use body for the request? To prevent logging the refresh token to appear in the 'usual' server logs.

EDIT: Nvm, I can see its supported either way.

Reply:
we can talk about it, but we are supporting both https://github.com/AckeeCZ/authist/pull/16/files#diff-92297ffb760d9441110d4820acd62f62c6ab5eb58edd9c3a6783f321084b37d6R64 `body` and `query` - for example facebook and google are using the access token in query too